tests/manage/test_ocs_324.py

```diff
@@ -92,11 +94,24 @@ def test_pvc_delete_create_same_name(self, test_fixture):
         global PVC_OBJ
 
         PVC_OBJ = helpers.create_pvc(sc_name=self.sc_obj.name)
-        logger.info(f"Deleting PersistentVolumeClaim with name {PVC_OBJ.name}")
+        pv_obj = ocp.OCP(
+            kind=constants.PV, namespace=config.ENV_DATA['cluster_namespace']
+        )
+        backed_pv = PVC_OBJ.get().get('spec').get('volumeName')
+        pv_status = pv_obj.get(backed_pv).get('status').get('phase')
+        assert constants.STATUS_BOUND in pv_status, (
+            f"{pv_obj.kind} {backed_pv} failed to reach {constants.STATUS_BOUND}"
+        )
+
+        logger.info(f"Deleting {PVC_OBJ.kind} {PVC_OBJ.name}")
         assert PVC_OBJ.delete(), f"Failed to delete PVC"
+
+        logger.info(f"Creating {PVC_OBJ.kind} with same name {PVC_OBJ.name}")
         PVC_OBJ = helpers.create_pvc(
             sc_name=self.sc_obj.name, pvc_name=PVC_OBJ.name
         )
-        logger.info(
-            f"PersistentVolumeClaim created with same name {PVC_OBJ.name}"
+        backed_pv = PVC_OBJ.get().get('spec').get('volumeName')
```

Comment:
backed_pv = PVC_OBJ.backed_pv

Reply:
@RazTamir I'm using helpers.py to create PVC. The PVC_OBJ here does not have 'backed_pv' attribute.
Am I missing something?

Reply:
ok.

+1 on the patch - the issue is with the helper function